.git-blame-ignore-revs

```diff
@@ -1,18 +0,0 @@
-# Since git version 2.23, git-blame has a feature to ignore
```

Comment:
you probably deleted this file by mistake

Reply:
In fact, I had some failed messages because of that file, so I intentionally removed it, but was a mistake : 

![image](https://user-images.githubusercontent.com/11873372/132005457-618e2e5b-85d6-43f2-9461-b18adcb01ab5.png)